vendor/enless-wireless/enless-profile-868.yaml

```diff
@@ -0,0 +1,52 @@
+# Vendor profile ID, can be freely issued by the vendor
+# This vendor profile ID is also used on the QR code for LoRaWAN devices, see
+# https://lora-alliance.org/sites/default/files/2020-10/LoRa_Alliance_Vendor_ID_for_QR_Code.pdf
+vendorProfileID: 716
```

Comment:
Could you make sure the `vendorProfileID` is unique for every profile? Also the value should not be the `vendorID`.